lib/hawkular/alerts/alerts_api.rb

```diff
@@ -20,6 +20,13 @@ def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials
       super(entrypoint, credentials, options)
     end
 
+    # Return version and status information for the used version of Hawkular-Alerting
+    # @return [Hash{String=>String}]
+    #         ('Implementation-Version', 'Built-From-Git-SHA1', 'status')
+    def fetch_version_and_status
```

Comment:
This PR is not for changing the API.
It is just to include a missing API into the same consistency naming used before.
The name of the method is not important here, it can be longer or shorter, the point is to add it into a consistency way.
A change of the API can follow this one.